src/main/kotlin/javabot/operations/URLTitleOperation.kt

```diff
@@ -41,17 +43,19 @@ class URLTitleOperation @Inject constructor(bot: Javabot, adminDao: AdminDao,
     private fun postMessageToChannel(responses: MutableList<Message>, titlesToPost: List<String>, event: Message) {
         val title = if (titlesToPost.size == 1) "title" else "titles"
         responses.add(Message(event, "${event.user.nick}'s $title: " +
-                titlesToPost.joinToString(" | ", transform = { s -> "\"$s\"" })))
+                titlesToPost.joinToString(" | ", transform = { s ->
+                    if (s.length > 250) {
+                        "\"${s.substring(0, 260)}...}\""
```

Comment:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/coerce-at-most.html

Reply:
I'm not sure how well this would apply, though, because the string is transformed to a new value if the length is over a certain amount, it's not just trimmed to a maximum value.

Reply:
Yeah, you're probably right there....